gateway/main/settings.py

```diff
@@ -270,6 +270,9 @@
 SETTINGS_TOKEN_AUTH_VERIFICATION_URL = os.environ.get(
     "SETTINGS_TOKEN_AUTH_VERIFICATION_URL", None
 )
+# verification fields to check when returned from auth api
+# Example of checking multiple fields:
+#   "SETTINGS_TOKEN_AUTH_VERIFICATION_FIELD", "is_valid;some,nested,field"
```

Comment:
Could we add the example JSON to the comment here too? I found it helpful when trying to understand how to properly format the string for this field.

Reply:
Good suggestion!

Added in https://github.com/Qiskit-Extensions/quantum-serverless/pull/1059/commits/a720041c2280b64bd39227f60ff86cf084754678